native/ex_tokenizers/src/models.rs

```diff
@@ -378,7 +378,7 @@ pub fn models_unigram_init(
     };
 
     Ok(ExTokenizersModel::new(
-        tokenizers::models::unigram::Unigram::from(vocab, unk_id)?,
+        tokenizers::models::unigram::Unigram::from(vocab, unk_id, false)?,
```

Comment:
This param is for the spm fallback byte trick (see [here](https://github.com/huggingface/tokenizers/issues/929)). I've hardcoded it to false, which is the default, but we could expose it as an option.

Reply:
We have an option on BPE, so we can expose it here too, yeah :)